application/README.md

```diff
@@ -45,6 +45,9 @@ With this configuration, you can use the ent cli (https://dev.entando.org/next/d
 * Removing the src/main/docker/keycloak-db directory will result in the realm from src/main/docker/realm-config being
   reloaded on the next restart.
 * Access controls are applied both in the UI and in the microservices. Individual REST APIs (e.g. ProjectResource) must either have tight constraints on an admin or support user (via an `@PreAuthorize annotation`) or customer or project-level access checks (e.g. `projectService.checkProjectAccess(projectId)`).
+* For local installations, the '.env.local.template' must be renamed to  '.env.local' to initialize the FE properly.
```

Comment:
Minor change, "copied". If they rename it, that would be an update to the source code as far as git is concerned.